Explain limits on making too many concurrent updates to the same ruleset (#10145)
cloudflare/cloudflare-docs: content/ruleset-engine/rulesets-api/_index.md

## Get started

To get started, review the Rulesets [JSON object](/ruleset-engine/rulesets-api/json-object/) and the available [Endpoints](/ruleset-engine/rulesets-api/endpoints/).

> ## Limits

Reviewer: ```suggestion
---

## Limits
```

> Concurrent updates to the same ruleset should be avoided and there are rate limits in place to prevent the same ruleset being concurrently updated too many times. The exact limits depend on the size of the ruleset and volume of requests, and so can be different for each ruleset.
> 
> This limit is most frequently hit when trying to concurrently modify several rules in the same ruleset. To avoid this, it is recommended to instead [update the entire ruleset at once](/ruleset-engine/rulesets-api/update/).

Reviewer: ```suggestion
You should avoid making concurrent updates to the same ruleset. There are rate limits in place to prevent the same ruleset from being concurrently updated too many times. The exact limits depend on the size of the ruleset and volume of requests, and can be different for each ruleset.

The rate limits are most frequently hit when concurrently modifying several rules in the same ruleset. To avoid this, you should [update the entire ruleset in a single operation](/ruleset-engine/rulesets-api/update/) instead.
```
